index.html

```diff
@@ -175,8 +235,9 @@ <h2 class="about-header">About me</h2>
               <button class="button" type="button">Get My Resume</button>
             </div>
           </div>
-            <hr class="separate" />
-            <div>
+         <div class="img-line-div"> <img src="./images/Devider.png" class="divider" alt="divider"> </div>
```

Comment:
- [ ] For the `about me` section Please add the right font family and font-weight to all heading text and try to stick as much as possible to the given Figma design.
- [ ] Kindly add the proper margin on the left to match the design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159835375-b2989924-47be-42ef-830d-fd50f4d767d3.png) | ![image](https://user-images.githubusercontent.com/68796232/159835272-84caa166-756a-4e98-9be8-dcad2bf0fc5d.png) |